.php_cs.dist

```diff
@@ -6,18 +6,13 @@ if (!file_exists(__DIR__.'/src')) {
 
 return PhpCsFixer\Config::create()
     ->setRules([
+        '@PHP71Migration' => true,
+        '@PHPUnit75Migration:risky' => true,        
         '@Symfony' => true,
         '@Symfony:risky' => true,
-        '@PHPUnit75Migration:risky' => true,
-        'php_unit_dedicate_assert' => ['target' => '5.6'],
-        'array_syntax' => ['syntax' => 'short'],
-        'fopen_flags' => false,
         'protected_to_private' => false,
         'native_constant_invocation' => true,
```

Comment:
enabling the rules as-is is causing the rule to use `strict => false` setting (ref https://cs.symfony.com/doc/rules/constant_notation/native_constant_invocation.html#strict )

when rule was introduced, on request of Sf, we set up `strict => true` for Symfony ruleset.
if you prefer `strict => false`, we can adjust the ruleset definition

Reply:
we now decided to always add the leading `\` to all constants

Reply:
ack, thanks for confirmation

Reply:
measurable perf difference if I recall well